ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -94,9 +116,20 @@ class IdpLoginScreen extends React.Component<Props, State> {
     }
     const loginUri = LOGIN_BASE_URL + loggedOutWithIdpAuth.idp.entityID;
 
+    const handleOnError = (): void =>
+      this.setState({
+        requestState: pot.noneError("error")
+      });
+
+    const goBack = this.props.navigation.goBack;
+    const setRequestStateToLoading = () =>
+      this.setState({ requestState: pot.noneLoading });
+
     const handleNavigationStateChange = (event: NavState): void => {
       this.setState({
-        isWebViewLoading: event.loading ? event.loading : false
+        requestState: event.loading
+          ? pot.noneLoading
```

Comment:
Are you sure about this? Why you are sure it is noneLoading and not someLoading?

Reply:
@fpersico yes, someLoading has more sense when webview changes state